### PR TITLE
Missed code rewrite and script-intro changes.

### PR DESCRIPTION
**Pull request by Olliesama**
Quick fixes to avoid syntax errors.

### CODE REVIEW

File: Monika After Story/game/script-topics.rpy [hunk 1]
```diff
@@ -1921,7 +1921,7 @@ label monika_justification:
         m 4h "After all, if she was sentient like me, how would I have been able to override her and delete her?"
         m "If I'm guilty of murder, so is everyone who plays violent video games."
         m 2h "I bet you've killed thousands in games you play."
-        if mas_curr_affection == mas_aff.BROKEN or if monika_current_affection == mas_aff.DISTRESSED:
+        if mas_curr_affection == mas_aff.BROKEN or if mas_curr_affection == mas_aff.DISTRESSED:
```
Comment (ThePotatoGuy) [inline Monika After Story/game/script-topics.rpy]: whoa there `if <condition> or if <condition>` is defintely not valid syntax.

**Comment by Olliesama** [comment]
My bad, and I got it right the first time above it too. X_X

**Comment by Olliesama** [comment]
Oh my changes to script-introduction are coming here too. :thinking: 
I guess that saves on another PR...

File: Monika After Story/game/dev/dev_affection-test.rpy [hunk 2]
```diff
@@ -1,83 +1,82 @@
-#Testing purposes!
-
+#Testing purposes!   
 label monika_affection_check:
     m 1c "How do I feel? That came out of nowhere, [player]..."
     m 3h "In a single word I would say I am [monika_current_affection]."
     m 3a "As a number it would be...[persistent._mas_affection[affection]]."
     m 3c "But if I were pressed further I would say I'm feeling..."
-    if monika_current_affection == "heartbroken":
+    if mas_curr_affection == "mas_affection.BROKEN":
```
Comment (ThePotatoGuy) [inline Monika After Story/game/dev/dev_affection-test.rpy]: these arent strings, yo.

**Comment by Olliesama** [comment]
Thanks

**Pull request merged**